tests/tool_use/utils.py

```diff
@@ -46,6 +46,7 @@ def ensure_system_prompt(messages: list[dict[str, Any]],
         "model":
         "NousResearch/Hermes-3-Llama-3.1-8B",
         "arguments": [
+            "--enforce-eager", "--no-enable-prefix-caching",
```

Comment:
- use `--enforce-eager` to make the tests faster
- use `--no-enable-prefix-caching` as numerics were causing tests to fail

Reply:
Is there an issue about the failure so it’s tracked somewhere? I assume we don’t want to keep this option here long term?